SPECS/subversion/subversion.spec

```diff
@@ -0,0 +1,55 @@
+Summary:    	The Apache HTTP Server
```

Comment:
Subversion file is not found on bintray!

From: Touseef Liaqat <notifications@github.com<mailto:notifications@github.com>>
Reply-To: vmware/photon <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, July 14, 2015 at 1:01 PM
To: vmware/photon <photon@noreply.github.com<mailto:photon@noreply.github.com>>
Subject: Re: [photon] Update for mesos (#186)


In SPECS/subversion/subversion.spec<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186-23discussion-5Fr34612367&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=-haYzmx278tKb9Xx1ELbeFxCo1wTihwGUppL0Nz2QtE&m=K2wti2NvB_2gWYc-CbgmjLPiZw1-bT2h1KW2QpG91bA&s=J-obmAj_YwTsPyerqu-hszsC7TTmMq4tO6UggH746jE&e=>:

> @@ -0,0 +1,55 @@
> +Summary:     The Apache HTTP Server


Fix Summary.

-
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186_files-23r34612367&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=-haYzmx278tKb9Xx1ELbeFxCo1wTihwGUppL0Nz2QtE&m=K2wti2NvB_2gWYc-CbgmjLPiZw1-bT2h1KW2QpG91bA&s=BIv4rWYOhDWGf587XZfKGall0Kpe5IozXo4WQz9ETyE&e=>.